src/settings-compat-v7/_color.scss

```diff
@@ -0,0 +1,11 @@
+// --- Colors --- //
+// This is a layer to maintain compatiblity with version 7-
+
+// Colors non-existing in the new color palette
+$c-highlighted: #f2cd96 !default; // seem not used
+$c-featured: #ffe613 !default; // used as category tag
+$c-white: #ffffff !default; // not semantic
```

Comment:
We're using `$c-white` and `$c-black` in Fotocasa even though they're naming conventions might be a lot better than that.
My proposal is to follow the convention by setting these values to `$c-lightest` and `$c-darkest`

Reply:
For now, we won't. We'll discuss to add it later on. For now, we strictly set values defined by design.